### PR TITLE
Add Alibaba DLA blog

### PR DESCRIPTION
**Pull request by fgwang7w**
Data Lake Analytics: Alibaba's Federated Cloud Strategy

### CODE REVIEW

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: Data Lake Analytics: Alibaba's Federated Cloud Strategy
+author: George Wang
+authorURL: https://www.linkedin.com/in/george-wang-9a5a46190/
+---
+
+
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:10]: Can you refactor this enough that there is an index and then add the truncate tag (you can see it in other blog posts).

This will prevent the entire blog post from appearing in the index. It will only show everything above the truncate tag.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+<br><br>
+
+**Second Layer**: Cloud native data lake analysis service layer.
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.
+<br>
+1. Elasticity- which allows flexibility capability meaning on demand, flexible in time, predictable and intelligent mixed load handling ability
+2. Resiliency - User’s unperceived High Availability capability, including transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:25]: Maybe shorten and rephrase this to

"High Availability capability, including transparent rolling upgrade, fast failover, and disaster tolerance across AZ (available area), etc"

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+
+## Data Lake Analytics
+### Introducing DLA 
+
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud.
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:53]: Is it forked from 0.227 permanently or is it continuing to track future updates to PrestoDB?

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+
+### Typical Data Flow Scenarios:
+
+Generally DLA empowers cloud users analytic capability in 3 typical user data flow scenarios:<br>
+1. The cloud user uploads the business generated data such as Log, CSV, JSON and other files, directly to OSS(Alicloud Object Storage Service), and then uses DLA to directly point to the file or folder for which the table is created to query. Then the user may use BI tools to analyze and visualize business data analytics.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1.png)
+<br><br>
+2. The cloud user can directly copy and upload data in the Hadoop’s legacy format, such as Parquet, ORC, RCFile, Avro, and other formats, to OSS, then use DLA to directly point files or folders for which the table is created to query. Then the user may use BI tools for business data analysis and visualization. <br>
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:92]: Are all of the formats listed legacy or just some? Because 3 suggests using Parquet or ORC as if they aren't legacy.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+3. In order to provide better query performance and low storage cost for subsequent data analysis on OSS, data can be converted into Parquet or ORC format to improve the cost performance of repeated data analysis.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case3.png)
+<br><br>
+
+### Alibaba Use Case Studies: 
+
+#### Customer Case #1: Log Data Analytics
+
+Cloud customers generate log data which is stored on OSS. The challenge is that those log data are unstructured and typically they are in Petabytes of data size. <br>
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:105]: Maybe
"The challenge is that the log data is unstructured and typically on the order of petabytes in size."

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1_logAnalysis.png)
+<br><br>
+So in order to make it available to analyze, DLA’s massive parallel processing engine is the key to business success. It allows users to do full-volume, personalized log data analysis by using DLA’s Hive connector. Additionally log data can be synchronized to OSS with a max delay of 5 minutes with Alibaba Cloud SLS that can feed data to DLA in near real-time.
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data are stored in various data sources. Its raw data containing all bioinformatics for storing gene sequence variations are stored as Variant Call Format(VCF). The challenge is that the Genetic datasets are huge and need to be analyzed promptly.
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:115]: Maybe

"The data volume is gigantic and the data is stored in various data sources. This is raw bioinformatics data storing gene sequence variations in the Variant Call Format (VCF). The challenge is that genetic datasets are huge, but still need to be analyzed promptly."

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data are stored in various data sources. Its raw data containing all bioinformatics for storing gene sequence variations are stored as Variant Call Format(VCF). The challenge is that the Genetic datasets are huge and need to be analyzed promptly. 
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2_gene_data_analytics.png)
+<br><br>
+The solution is to place a huge amount of gene data on OSS for low cost storage service. In the meantime, its reference data that are created as indexing data are stored in the KV system in a distributed NoSQL database by Aliyun Open Table Service(OTS). Now the user may use DLA’s cross data source analytic capability to join the data together to quickly access and analyze for which the resulting query runtime is promising.
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:119]: Suggested rewrite for this paragraph:

The solution is to place the high volume gene data on OSS where storage costs are low and materialize indexing data to a KV system. Aliyun Open Table Service (OTS) is a distributed NoSQL database that works well for this purpose.

DLA's cross data source analytic capabilities can then be used to join the data together across services making it possible to quickly access and analyze the data.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+<br><br>
+
+
+# Conclusion
+
+
+Alibaba Data Lake Analytics embraces Presto’s federated query engine capability and has empowered many Alibaba cloud users to experience large scale serverless cloud service. DLA is working to make cloud user experience more transparent for ease of use and ease of management. The next blog will discuss some key technical development works on top of Presto codebase.
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:126]: Suggestion:

DLA is working to make the cloud user experience more transparent by improving ease of use and reducing infrastructure management overhead. The next blog will discuss some key technical developments on top of the Presto codebase.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
+
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/threeLayerCloud_v1.png)
+<br><br>
+**First Layer**: A variety of selected cloud native data storage and database services. 
+<br><br>
+Users store and place their data on a series of cloud-native data storage services and cloud databases systems. Cloud data sources vary depending on different business use cases and business scenarios, but in general speaking, there are three cloud data storage: NoSQL and relational databases. Various choices of storage service may include: cost-effective object cloud storage which may store files, structured, semi-structured, unstructured raw data, multimedia files; NoSQL, RDS, and other cloud database services;
```
[the remark this answers is not in the document]
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:17]: yes fixed, many thanks

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 1 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:10]: got it, I have revised by adding an short intro with index paragraph

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 2 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:25]: so for this I'd like to revised a bit by putting the same pattern because it should be clear to user what are the key component features offered in this second layer

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 3 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:53]: we do plan to stay close with the community with a close contact with the community to understand the latest updates and ongoing projects so we can pickup important features from the community or contribute any bugfix or features back to the PrestoDB.

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 5 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:105]: fixed, thanks

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 6 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:115]: fixed, thanks

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 7 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:119]: fixed, made a minor different at the end of the sentense to point out the importance of the runtime vs. large sizing scale of datasets

**Comment by fgwang7w** [comment]
> The introduction needs a little work so we can add a truncate tag so the entire blog doesn't appear in the index.
> 
> I also suggested some rewrites for clarity.
> 
> I am also wondering if this forked at 0.227 permanently or if the changes are tracking Presto over time.

many thanks for your valuable feedback, i believe current revised version looks much more promising than before given the new indexing header, minor content revision, and final wording. Please help to review again if possible to let me know if it's a green light to go on, cheers!

**Comment by fgwang7w** [comment]
> @fgwang7w Thanks for your contribution. 👍 Can you please make edits required per @aweisberg?


Thanks for your review, revisions are done in the new commit now.cheers!

**Comment by fgwang7w** [comment]
hi all, could you please review the doc again for merge? thanks! @aweisberg @aweisberg @nezihyigitbasi

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+<br>
+Alibaba Data Lake Analytics embraces Presto’s federated query engine capability and has accumulated a number of successful business use cases that signify the power of Presto's analytics capability.
+<br><br>
+
+
+<!-- truncate -->
+
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:17]: "Today’s cloud databases empower"

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/threeLayerCloud_v1.png)
+<br><br>
+**First Layer**: A variety of selected cloud native data storage and database services. 
+<br><br>
+Users store and place their data on a series of cloud-native data storage services and cloud databases systems. Cloud data sources vary depending on different business use cases and business scenarios, but speaking in general, there are two cloud data storage: NoSQL and relational databases. Various choices of storage service may include: cost-effective object cloud storage which may store files, structured, semi-structured, unstructured raw data, multimedia files; NoSQL, RDS, and other cloud database services;
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:24]: "there are two cloud data storage" reads weird. Maybe "data storage approaches".

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+<br><br>
+
+**Second Layer**: Cloud native data lake analysis service layer.
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.<br>
+
+1. Elasticity - This allows an on-demand flexibility capability, which means being flexible in time, predictable and intelligent mixed load handling ability<br>
+
+2. Resiliency - User’s unperceived High Availability capability includes transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:33]: "unperceived High Availability capability" => not clear what this means. Can you please update/revise?

"available area" => availability zone

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+
+3. Federated and Analytical - Multi-mode federal analysis capabilities, includes analysis and integration capabilities for multiple formats and systems, comprehensive analysis functions and excellent interactive analysis performance and experience, parallel computing processing capabilities, compatible interface capability and so on.<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:35]: "federal analysis" => federated analysis ?

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+
+## Data Lake Analytics
+### Introducing DLA 
+
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud. It is currently working closely with the PrestoDB community to track ongoing projects and updates. It is mission critical to continue the close partnership with PrestoDB for business success with latest updates.
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:59]: > Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users.

This reads as MySQL becoming a connection protocol. I guess you mean sth like MySQL has a user friendly connection protocol. Please revise.

Comment (nezihyigitbasi) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:59]: "ALL metadata" => all metadata (don't need caps)

Comment (nezihyigitbasi) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:59]: "PrestoDB Version 0.227" => PrestoDB (v0.227)

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+<br><br>
+
+### DLA Product Key Features:
+
+1. Serverless:<br>
+a. No infrastructure and management cost<br>
+b. Zero start-time<br>
+c. Transparent upgrade<br>
+d. QoS Elastic Service<br>
+2. Standard SQL and Connections<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:69]: What does "Connections" refer to?

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 16]
```diff
@@ -0,0 +1,132 @@
...
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud. It is currently working closely with the PrestoDB community to track ongoing projects and updates. It is mission critical to continue the close partnership with PrestoDB for business success with latest updates.
+<br><br>
+
+### DLA Product Key Features:
+
+1. Serverless:<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:64]: nit: the other numbered items don't have ":" in the end, so please remove this one as well to be consistent.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 17]
```diff
@@ -0,0 +1,132 @@
...
+a. No infrastructure and management cost<br>
+b. Zero start-time<br>
+c. Transparent upgrade<br>
+d. QoS Elastic Service<br>
+2. Standard SQL and Connections<br>
+a. Compatible with standard SQL<br>
+b. Rich built-in function support<br>
+c. JDBC/ODBC support<br>
+d. Compatible with BI tools<br>
+3. Heterogeneous Data Source<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:74]: "Heterogeneous Data Sources" (plural)

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 18]
```diff
@@ -0,0 +1,132 @@
...
+a. Enable OSS Data analysis requirements<br>
+b. Enable Table Store Data for SQL querying service<br>
+c. Federated data analysis across multiple database instances<br>
+d. Ease of interconnection analysis for multiple data sources<br>
+e. Any forms of analytics in “Data Lake”<br>
+4. Optimized Computing Engine<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:80]: "Compute Engine"

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+a. Support for unstructured data<br>
+b. Vectorized execution and optimization<br>
+c. Operator pipeline optimization<br>
+d. Resource isolation and priority<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:84]: "priority" => prioritization ?

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 20]
```diff
@@ -0,0 +1,132 @@
...
+
+
+
+## DLA Use Cases
+
+
+### Typical Data Flow Scenarios:
+
+Generally DLA empowers cloud users analytic capability in 3 typical user data flow scenarios:<br>
+1. The cloud user uploads the business generated data such as Log, CSV, JSON and other files, directly to OSS(Alicloud Object Storage Service), and then uses DLA to directly point to the file or folder for which the table is created to query. Then the user may use BI tools to analyze and visualize business data analytics.<br>
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:94]: space before parens.

"OSS(" => "OSS ("

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 21]
```diff
@@ -0,0 +1,132 @@
...
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data is stored in various data sources. This is raw bioinformatics data storing gene sequence variations in the Variant Call Format(VCF). The challenge is that Genetic datasets are huge, but still need to be analyzed promptly. 
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2_gene_data_analytics.png)
+<br><br>
+The solution is to place a huge volume gene data on OSS where storage costs are low and materialize indexing data to a KV system. Aliyun Open Table Service(OTS) is a distributed NoSQL database that works well for this propose. DLA's cross data source analytic capabilities can then be used to join the data together across services making it possible to quickly access and analyze the data for which the resulting query runtime is very promising for users when running on large-sizing scale of datasets.
```
Comment (nezihyigitbasi) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:125]: KV system => key value store (reader may not know what KV abbreviation stands for)

Comment (nezihyigitbasi) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:125]: "Service (OTS)"

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 10 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:17]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 11 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:24]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 12 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:33]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 13 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:35]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 14 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:59]: yah the sentense is confusing, revised

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 17 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:74]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 18 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:80]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 19 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:84]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 20 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:94]: fixed

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 21 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:125]: fixed

**Comment by aweisberg** [comment]
@fgwang7w  You are marking feedback as resolved without commenting or changing anything. Some of these things are parts that both Nezih and myself have stumbled over.

I would like to get this published, but we really need to get the requested changes done in order to do that.

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 4 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:98]: Some of the lagacy file formats can be used by the DLA user.

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 8 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md]: fixed

**Comment by fgwang7w** [comment]
> @fgwang7w You are marking feedback as resolved without commenting or changing anything. Some of these things are parts that both Nezih and myself have stumbled over.
> 
> I would like to get this published, but we really need to get the requested changes done in order to do that.

Hi I believe I fixed all the comments with feedback. The revision had been pushed, i did squash into 1 commit, could you pleaes review again? @aweisberg Thanks

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 15 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:69]: should be SQL connections with an extra "and", fixed

**Comment by fgwang7w** [comment]
> @fgwang7w You are marking feedback as resolved without commenting or changing anything. Some of these things are parts that both Nezih and myself have stumbled over.
> 
> I would like to get this published, but we really need to get the requested changes done in order to do that.

I made another found of review and carefully went through each of the comments. I also added the truncate tag. please let me know if this version looks good for publish. Many thanks!

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 22]
```diff
@@ -0,0 +1,132 @@
...
+<br><br>
+
+#### Customer Case #2: Genetic Data Analytics
+
+
+One of Alicloud’s data vendors is in the Genetic engineering industry. The data volume is gigantic and the data is stored in various data sources. This is raw bioinformatics data storing gene sequence variations in the Variant Call Format(VCF). The challenge is that Genetic datasets are huge, but still need to be analyzed promptly. 
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2_gene_data_analytics.png)
+<br><br>
+The solution is to place a huge volume gene data on OSS where storage costs are low and materialize indexing data to a Key value store system. Aliyun Open Table Service (OTS) is a distributed NoSQL database that works well for this propose. DLA's cross data source analytic capabilities can then be used to join the data together across services making it possible to quickly access and analyze the data for which the resulting query runtime is very promising for users when running on large-sizing scale of datasets.
```
Comment (aweisberg) [inline website/blog/2020-06-30-data-lake-analytics-blog.md:125]: "quickly access and analyze the data for which the resulting query runtime is very promising for users when running on large-sizing scale of datasets."
->
"quickly access and analyze the data even on large scale datasets"

It's much shorter and says all the same things. "Very promising" doesn't mean that the result was good it means it was potentially good. Quickly means it was explicitly fast.

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 23]
```diff
@@ -0,0 +1,132 @@
...
+<br><br>
+
+**Second Layer**: Cloud native data lake analysis service layer.
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.<br>
+
+1. Elasticity - This allows an on-demand flexibility capability, which means being flexible in time, predictable and intelligent mixed load handling ability<br>
+
+2. Resiliency - High Availability gives user an unperceived experience includes transparent rolling upgrade, fast handover, disaster tolerance across AZ (availability zones), etc<br>
```
[the remark this answers is not in the document]
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:33]: thank you for pointing this out, sorry for the confusion on the use of terminology, this is a typical disaster recovery procedure for high availbility systems. When failures occurs, the failover process moves processing performed by the failed component to the babckup componenet within a matter of microseconds. This failover becomes more transparent to users for a better user experience in a HA system. I will rewrite this point accordingly

File: website/blog/2020-06-30-data-lake-analytics-blog.md [hunk 24]
```diff
@@ -0,0 +1,132 @@
...
+
+
+## Data Lake Analytics
+### Introducing DLA 
+
+Data Lake Analytics, known as DLA, is a large scale serverless data federation service on Alibaba Cloud. It is one of the most popular serverless SQL engines based on a well-customized computing engine from PrestoDB. DLA integrates with mainstream data sources and provides easy-to-use MySQL JDBC/ODBC connection protocol to allow users to interact with.
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. MySQL is a flexible and compatible connection protocol service, frequently used by DLA users. Unified Meta Layer is a metadata repository which holds all metadata for all data sources DLA can support. DLA was forked from PrestoDB(0.227) and is now integrated with mainstream data sources in Alibaba Cloud. It is currently working closely with the PrestoDB community to track ongoing projects and updates. It is mission critical to continue the close partnership with PrestoDB for business success with latest updates.
```
[the remark this answers is not in the document]
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:59]: It was used the term "forked from" to indicate that DLA is running a new project which is indepedent from prestoDB. Not all prestoDB commits were pulled to downstream. Only partial commits that are related are being cherry-picked from prestoDB. So I will incorporate this in the sentense to make it clear to the user about this information.

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 16 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:64]: sorry missed, ":" is removed now

File: website/blog/2020-06-30-data-lake-analytics-blog.md (hunk 22 above)
Comment (fgwang7w) [reply website/blog/2020-06-30-data-lake-analytics-blog.md:125]: many thanks for the suggestion, it is revised per suggestion

**Comment by fgwang7w** [comment]
@aweisberg hi thank you so much for giving another round of review. I made sure all issues are properly addressed or rewritten. Hope it looks good to you now~

**Comment by shixuan-fan** [comment]
@aweisberg @dborkar  @nezihyigitbasi Any more comments on this? I think this is good to merge since there has been no more comments for quite a while.

**Comment by shixuan-fan** [comment]
If no objection, I'm going to merge this tomorrow :)

**Comment by dborkar** [comment]
@fgwang7w Thank you for the changes. LGTM. @shixuan-fan feel free to merge.

**Comment by shixuan-fan** [comment]
Since @nezihyigitbasi is on vacation and I got other two approvals. I'm going to merge it now :)

Thanks for your contribution @fgwang7w

**Comment by aweisberg** [comment]
One sanity check, is the date of the blog post still reasonable?

**Comment by aweisberg** [comment]
@shixuan-fan can you do a follow up change and set the date to today? The images and the directory the images are in also depend on this. 

Sorry :-(

If we don't update it then it will not be the top blog post.
